regression/solver-hardness/solver-hardness-simple/test.desc

```diff
@@ -0,0 +1,11 @@
+CORE
+main.c
+--write-solver-stats-to "solver_hardness.json"
+^EXIT=10$
+^SIGNAL=0$
+^\[main.assertion.\d+\] line \d+ assertion a \+ b \< 10: FAILURE$
+^VERIFICATION FAILED$
+^Valid JSON File$
+\"SAT_hardness\": \{(\"Clauses\": \d+|\"Variables\": \d+|\"Literals\": \d+), (\"Clauses\": \d+|\"Variables\": \d+|\"Literals\": \d+), (\"Clauses\": \d+|\"Variables\": \d+|\"Literals\": \d+)\}
```

Comment:
As far as tests go this isn’t very exciting, but I couldn’t figure out a good way to put specific values here either; Any changes to symex or solvers would invalidate any test that’s checking for concrete values here.

Reply:
You could check that there is something non-zero, at least.